astropy/modeling/rotations.py

```diff
@@ -192,8 +192,8 @@ def evaluate(self, alpha, delta, phi, theta, psi, axes_order):
         result = np.dot(matrix, inp)
         a, b = cartesian2spherical(*result)
         if shape is not None:
-            a.shape = shape
-            b.shape = shape
+            a = a.reshape(shape)
```

Comment:
This routine can be fairly easily rewritten to not do the ravel, reshape at all.  I can push a commit to your branch, though perhaps it is better I do it in follow-up, so we can just get this in first, since what you have here is the obvious solution of just replacing setting `.shape` with a `.reshape()`.

Reply:
Yes, I prefer a followup pR.